server/channels/api4/upload.go

```diff
@@ -40,6 +41,8 @@ func createUpload(c *Context, w http.ResponseWriter, r *http.Request) {
 	us.RemoteId = ""
 	us.ReqFileId = ""
 
+	us.Filename = filepath.Base(us.Filename)
```

Comment:
Interesting. Looks like we cleaned it for generating the final `UploadSession.Path`, but passed it through unchecked to `genFileInfoFromReader`